### PR TITLE
OWL-2825: Moved the delete button for Lessons

### PR DESCRIPTION
**Pull request by chaospeach**
https://jira.uwo.ca/browse/OWL-2825

Less likely to accidentally click on the delete button accidentally when editing Lessons Items

### CODE REVIEW

File: lessonbuilder/tool/src/webapp/css/Simplepagetool.css [hunk 1]
```diff
@@ -198,7 +198,7 @@ div.action span .nextbutton, .Mrphs-siteHierarchy span .nextbutton{
 
 }
 .act .deleteButton {
-    float: none
+    float: right
```
Comment (plukasew) [inline lessonbuilder/tool/src/webapp/css/Simplepagetool.css]: Is this going to be too far to the right on wide screens and easily missed by users looking for it?

Comment (fostersdesign) [reply lessonbuilder/tool/src/webapp/css/Simplepagetool.css]: Now that it looks like a red button, will that help/resolve your concern, @plukasew ?

Comment (plukasew) [reply lessonbuilder/tool/src/webapp/css/Simplepagetool.css]: I'm still not sure the far right is the correct place for this on wider screens, but I suppose that is more of a responsive design issue. I think it is certainly more visible as a button though.

File: lessonbuilder/tool/src/webapp/css/Simplepagetool.css [hunk 2]
```diff
@@ -190,20 +190,30 @@ div.action span .nextbutton, .Mrphs-siteHierarchy span .nextbutton{
...
 }
-.controlsPanel .act .deleteButton:hover, .act .deleteButton:hover {
+/*.controlsPanel .act .deleteButton:hover, .act .deleteButton:hover {
     background: none !important;
     cursor: pointer;
-}
+}*/
```
Comment (bjones86) [inline lessonbuilder/tool/src/webapp/css/Simplepagetool.css]: Can we just get rid of these comment blocks?

Comment (chaospeach) [reply lessonbuilder/tool/src/webapp/css/Simplepagetool.css]: Yep!

**Comment by chaospeach** [comment]
As discussed. The "Delete" link is now a button, and commented out code has been removed.

**Review by fostersdesign — CHANGES_REQUESTED** [review]
I like that the button is styled as a button now, but we don't have red delete buttons elsewhere in Sakai right now.

The correct fix options for this issue are:

1. add a confirmation prompt to the button in this panel similar to the confirmation on the main page (from the trash can icon)
2. or remove the delete button from the edit panel and make them use the trash can icon on the main page (similar to JIRA comments with the edit panel not having a delete button)

**Comment by chaospeach** [comment]
As per option 2. The button is now removed.

**Pull request closed without merging**